gdrive/settings.py

```diff
@@ -19,6 +19,8 @@
 ROOT_DIRECTORY = ""
 CREDENTIALS = None
 
+ES_HOST = os.getenv("ES_HOST", "localhost")
```

Comment:
does this need to stay localhost as default?

Reply:
That is what it falls back on when deployed locally. I can always set that locally another way. Should the default value be something else?

Reply:
I would rather leave it blank and set the env variable locally + catch a bad url for ES in the code